### PR TITLE
Add Contact form

### PR DESCRIPTION
**Pull request by Lasten-Ernest**
This pull request is for adding a contact form in the portfolio project. The contact form section is a collaboration work between me, Ernest and my partner, Dorian.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -253,6 +253,21 @@ <h2 class="media_header">LET'S CONNECT</h2>
         </div>
     </section>
     </div>
+    <footer>
+			<form action="https://formspree.io/f/xwkyyyvp" method="post">
+				<h2>Contact me</h2>
+				<p class="contact-infor">
+					If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it 
+				</p>
+				<input name="name" type="text" placeholder="Name">
```
Comment (ShahierNashaat) [inline index.html]: - [ ] Kindly be noticed that the name should have maxlength="30" and required attributes.

File: index.html [hunk 2]
```diff
@@ -253,6 +253,21 @@ <h2 class="media_header">LET'S CONNECT</h2>
...
+				<input name="email" type="email" placeholder="Email">
+				<textarea name="message" id="message" placeholder="Write your message here"></textarea>
```
Comment (ShahierNashaat) [inline index.html]: - [ ] Kindly be noticed that the message should have maxlength="500" and required attributes.

File: index.html [hunk 3]
```diff
@@ -253,6 +253,21 @@ <h2 class="media_header">LET'S CONNECT</h2>
...
     </section>
     </div>
+    <footer>
+			<form action="https://formspree.io/f/xwkyyyvp" method="post">
+				<h2>Contact me</h2>
+				<p class="contact-infor">
+					If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it 
+				</p>
+				<input name="name" type="text" placeholder="Name">
+				<input name="email" type="email" placeholder="Email">
```
Comment (ShahierNashaat) [inline index.html]: - [ ] Kindly be noticed that the email should have a required attribute.

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi @Lasten-Ernest and @dodoburner , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sja-thedude — APPROVED** [review]
## Project Approved 👍 

Hi @Lasten-Ernest, This is @sja-thedude your code reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

giphy

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**